popup/search.js

```diff
@@ -287,17 +294,11 @@
     const elShot = $('.search-result-screenshot-default', entry);
     if (isUsw) {
       if (/^https?:/i.test(shotName)) {
-        elShot.src = shotName;
-        // USw has by default a screenshot of the style in webp format.
-        // But for compatability reasons always deliver the jpg format.
-        // Webp format is more efficient and thus is preffered to be used.
-        // When supported.
-        const webpElShot = $create('source', {
-          type: 'image/webp',
-          srcset: shotName.replace(/\.jpg$/, '.webp'),
-        });
-        const pictureEl = $('.search-result-screenshot', entry);
-        pictureEl.insertBefore(webpElShot, pictureEl.firstChild);
+        if (imgType !== '.jpg') {
```

Comment:
Do you dislike ternary? :-) I think it would be better here, instead of three nested ifs.